indra/newview/character/avatar_constraint.llsd

```diff
@@ -43,12 +43,12 @@
         {
             'forward_axis':
                 [r-0,r-1,r-0],
-            'max_bend':r0,
-            'max_twist':r135,
-            'min_bend':r-157.5,
-            'min_twist':r-45,
+            'max_bend':r157.5,
+            'max_twist':r0.5,
+            'min_bend':r0.0,
+            'min_twist':r-0.5,
             'pivot_axis':
-                [r-0,r-0,r-1],
+                [r0,r0,r1],
             'type':'ELBOW'
         },
```

Comment:
I'm flipping the direction of the right elbow's pivot axis, and swizzling the bend limits accordingly.  This doesn't actually change the result, but it makes more sense to me when I try to visualize the constraint's geometry.